recompose-test/src/main/resources/edittext.xml

```diff
@@ -3,4 +3,6 @@
         xmlns:android="http://schemas.android.com/apk/res/android"
         android:layout_height="50dp"
         android:layout_width="200dp"
-        android:inputType="text" />
\ No newline at end of file
+        android:inputType="text"
+        android:hint="text"
```

Comment:
Can we create an additional test case here? Just so that we have both covered, with `hint` and without.